subxt/src/metadata/metadata_type.rs

```diff
@@ -304,7 +304,7 @@ pub struct EventMetadata {
     // behind an Arc to avoid lots of needless clones of it existing.
     pallet: Arc<str>,
     event: String,
-    fields: Vec<(Option<String>, u32)>,
+    fields: Vec<(Option<String>, Option<String>, u32)>,
```

Comment:
I think the introduction of a `struct` here e.g. `EventFieldMetadata` to disambiguate the fields would be good.